src/Frame.cpp

```diff
@@ -935,6 +938,55 @@ std::shared_ptr<QImage> Frame::GetImage()
 	return image;
 }
 
+#ifdef USE_OPENCV
+
+// Convert Qimage to Mat
+cv::Mat Frame::Qimage2mat( std::shared_ptr<QImage>& qimage) {
+
+    cv::Mat mat = cv::Mat(qimage->height(), qimage->width(), CV_8UC4, (uchar*)qimage->bits(), qimage->bytesPerLine()).clone();
+    cv::Mat mat2 = cv::Mat(mat.rows, mat.cols, CV_8UC3 );
+    int from_to[] = { 0,0,  1,1,  2,2 };
+    cv::mixChannels( &mat, 1, &mat2, 1, from_to, 3 );
+	cv::cvtColor(mat2, mat2, cv::COLOR_RGB2BGR);
+    return mat2;
+};
+
+// Get pointer to OpenCV image object
+cv::Mat Frame::GetImageCV()
+{
+	// Check for blank image
+	if (!image)
+		// Fill with black
+		AddColor(width, height, color);
+	
+	// if (imagecv.empty())
+	// Convert Qimage to Mat
+	imagecv = Qimage2mat(image);
+
+	return imagecv;
+}
+
+std::shared_ptr<QImage> Frame::Mat2Qimage(cv::Mat img){
+	cv::cvtColor(img, img, cv::COLOR_BGR2RGB);
+	QImage qimg((uchar*) img.data, img.cols, img.rows, img.step, QImage::Format_RGB888);
```

Comment:
We are using `QImage::Format_RGBA8888_Premultiplied` as our primary color format now, let's change that here